src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,32 @@
+package duke.parser;
+
+import duke.exceptions.UnknownCommandException;
+import duke.utils.Command;
+
+/**
+ * Parse input.
+ */
+public class Parser {
+    /**
+     * Splits input into substrings.
```

Comment:
Consider adding an empty line between the description and the parameter section? I noticed this minor issue in other areas of the code as well